aiida_vasp/calcs/immigrant.py

```diff
@@ -50,15 +49,29 @@ class VaspImmigrant(VaspCalculation):
 
        code = Code.get_from_string('vasp544mpi@local')
        folder = '/home/myself/vasp-calc-dir'
+       settings = {'parser_settings': {'add_energies': True,
```

Comment:
Can we leave these out such that the user have to set this in the script/call to the workchain? This is how we do it for the other workchains. Only parse what is strictly necessary. For the immigrant, we do not really know what the user wants. Some wants energy and force, some maybe just to check if the run completed and do a restart and so forth.

Reply:
This is an example. So the usage is the same as the other workchains. We will write more details in documentation.

Reply:
Great. I think we do `vasp@mycluster` in the tutorials.